src/modules/AutoinstScripts.rb

```diff
@@ -227,7 +227,7 @@ def Export
     def Resolve_ws(script)
       script = deep_copy(script)
       if !Builtins.isempty(Ops.get_string(script, "location", ""))
-        l = CutWhitespace(Ops.get_string(script, "location", ""))
+        l = Ops.get_string(script, "location", "").strip
```

Comment:
What about l = script.fetch("location", "").strip
I have learned this from Imo :-)